server/routes/river-and-sea-levels.js

```diff
@@ -31,9 +32,16 @@ module.exports = [{
       try {
         place = await locationService.find(util.cleanseLocation(location))
       } catch (error) {
-        // If location search error show national list with error
-        stations = await floodService.getStationsWithin([-6.73, 49.36, 2.85, 55.8])
-        model = new ViewModel({ location, place, stations, error })
+        console.error(`Location search error: [${error.name}] [${error.message}]`)
+        if (error instanceof LocationNotFoundError) {
+          // No location found so display zero results
+          stations = []
+          model = new ViewModel({ location, place, stations })
+        } else {
+          // If location search error show national list with error
+          stations = await floodService.getStationsWithin([-6.73, 49.36, 2.85, 55.8])
```

Comment:
I think this may be left over tech debt, but really we should have a service call here that just gets all stations, rather than the hack that geographically gets them